src/kOS/Suffixed/Timespan.cs

```diff
@@ -94,6 +94,7 @@ public double ToUnixStyleTime()
         public static TimeSpan operator -(TimeSpan a, double b) { return new TimeSpan(a.ToUnixStyleTime() - b); }
         public static TimeSpan operator *(TimeSpan a, double b) { return new TimeSpan(a.ToUnixStyleTime() * b); }
         public static TimeSpan operator /(TimeSpan a, double b) { return new TimeSpan(a.ToUnixStyleTime() / b); }
+        public static TimeSpan operator +(double b, TimeSpan a) { return new TimeSpan(b + a.ToUnixStyleTime()); }
```

Comment:
Well it supported the opposite order.  So `print 5 + time.` would display: "5TIME(...)" instead of adding 5s to the current time.  Just one of those fun order of operations issues.